utils/images.py

```
@@ -101,4 +133,57 @@ def generate_images_list(version, build):
+    images_list = []
+
+    try:
+        # 1. Create the container
+        # We use {{ }} to escape curly braces for awk in f-strings
+        get_images_command = (
+            f"podman create --name {container_name} "
+            f"$(opm alpha list bundles {url} | grep {version} | awk '{{print $6}}' | "
+            f"sed 's/registry.redhat.io/registry.stage.redhat.io/') /bin/true"
+        )
+
+        image_id, error = run_command(get_images_command)
```

Ruff (0.15.0)

[warning] Unpacked variable `image_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)